Engine/Source/Timer.h

```diff
@@ -3,24 +3,29 @@
 
 class Timer {
 public:
-	Timer() = default;
+    Timer() = default;
 
-	void Start();
-	void Update(); //Log frames and hault to match max FPS
+    void Start();
+    void StartWithRunTime();
 
-	void StartWithRunTime();
+    void Update(); //Logs frames and statistics and haults the application to match max FPS
 
-	long Read();
-	long ReadDelta();
+    long Read();
+    long ReadDelta();
 
-	long Stop() { SetSpeed(0); }
-	long SetSpeed(float speed);
-    void SetTimeScale(float speed);
+    long Stop();
+    void Pause() { SetTimeScale(0.f); }
```

Comment:
I'd propose to set all the methods in the cpp, cause we're using those to extend the functionality of each method. This would make things cleaner IMO.